apps/files/src/components/AllFilesList.vue

```diff
@@ -37,6 +37,16 @@
               {{ formDateFromNow(item.mdate) }}
             </div>
           </template>
+          <template #noContentMessage>
+            <no-content-message v-if="!$_isFavoritesList" icon="folder">
+              <template #message><translate>There are no resources in this folder.</translate></template>
```

Comment:
Why not using the slot directly on `translate` tag?

Reply:
Do you mean putting v-translate on the template tag ? I wasn't sure whether this works.
I guess I could test it with a dummy temporary local translation

Reply:
Not sure if that works. I thought more about `<translate v-slot:message>`

Reply:
Ah, now I see this was deprecated in 2.6.0. My bad...